devlog/2024-11-20-clean-cleanl-clean.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Clean, clean, clean!
+slug: clean-clean-clean
+authors: alfi
+tags: [housekeeping, cleanup, devlog, solus]
+hide_table_of_contents: false
+---
+
+Hello everyone! Alfi here, member of Solus Cleanup Crew. It's been more than a year since I started contributing to Solus and I want to talk about it. About what I did, what we did, what we accomplished so far, and what we want to do next.
+
+<!-- truncate -->
+
+## A bit of background
+I started contributing on October 2023, when Solus participate in [Hacktoberfest](https://getsol.us/2023/10/01/solus-and-hacktoberfest-2023/). Hacktoberefest is a month-long event that encourage people to contribute to open source project in exchange for some digital merchandise. Finally, an excuse to contribute to something I use everyday. Although, as an accountant, all of this is new experience to me. It was confusing at first, but I persisted. Finally, I get the hang of it and I managed to submit around 100 pull request during the month of the event. I was very excited and motivated. During the month, a few people joined the event and contributed to Solus, but then November came and most of the contribution slowed down. I was the only one left that continue doing the beginner tasks, mostly adding homepage to packages.
+
+At the start January 2024, Jakub Nugen ([androidnisse](https://github.com/androidnisse)) comes in and start contributing to homepage task too. Worried that we would stepped on each other works, we started a private chat to organize the tasks between ourselves. Until May, the two of us have been chipping away the homepage task which at this point leaving 650 packages without homepage. Then we get more contributors joining. I decided to make a group chat to accomodate all the people joining. It was called "Solus Homepage Team" at the time. More contributors and staff members join the room, then it was renamed as "Solus Cleanup Crew" and moved under official Solus matrix space.
+
+## What is it?
+Solus cleanup crew is a group of bunch of community contributors as well staff members with the goal to bringing up Solus package repository up to current standard. It is a subset of the packaging room on our matrix channel. We organize the work on all the cleanup related tasks.
+
+## Why bother?
+Solus has been updating their tooling and infrastucture since its resurection. While most of the tooling changes can be accomplished by staff members alone, there are things that they cannot do. While many consider Solus repository as small, it still contains more than 6000 packages. Some packages has never even got an update since Solus founded! We want to get rid of all its cruft before we can migrate to a better tooling. That's where the community contributor can come in and help.
```

Comment:
```suggestion
Solus has been updating their tooling and infrastucture since its resurection. While most of the tooling changes can be accomplished by staff members alone, there are things that they cannot do. While many consider Solus repository as small, it still contains more than 6000 packages. Some packages have never even gotten an update since Solus was founded! We want to get rid of all its cruft before we can migrate to a better tooling. That's where the community contributor can come in and help.
```
 Probably better to use a different phrase than "since Solus was founded". Maybe "since Solus was first created", or something.